nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/resources/templates/commentUpdated.ftl

```diff
@@ -0,0 +1,3 @@
+<#assign action = "updated">
+<#assign subscriptionName = "Modification">
```

Comment:
This will end up in the email as:
```
You received this notification because you subscribed to 'Modification comment' notification on this document or on one of its parents.
```
I find it weird... what about just 'Updated comment'?

Reply:
@aescaffre could have a better idea for the mail content :)